packages/opentelemetry-test-utils/README.md

```diff
@@ -65,6 +65,42 @@ it('some test', () => {
+| container.image.name    | Name of the image the container was built on.                                                 |
+| container.image.tag     | Container image tag.                                                                          |
+| container.name          | Container name.                                                                               |
+| host.id                 | Unique host ID.                                                                               |
+| host.image.id           | VM image ID.                                                                                  |
+| host.image.name         | Name of the VM image or OS install the host was instantiated from.                            |
+| host.image.version      | The version string of the VM image.                                                           |
+| host.name               | Name of the host.                                                                             |
+| host.type               | Type of host.                                                                                 |
+| k8s.cluster.name        | The name of the cluster.                                                                      |
+| k8s.deployment.name     | The name of the Deployment.                                                                   |
+| k8s.namespace.name      | The name of the namespace that the pod is running in.                                         |
+| k8s.pod.name            | The name of the Pod.                                                                          |
+| process.command         | The command used to launch the process (i.e. the command name).                               |
+| process.command_line    | The full command used to launch the process as a single string representing the full command. |
+| process.executable.name | The name of the process executable.                                                           |
+| process.pid             | Process identifier (PID).                                                                     |
+| service.instance.id     | The string ID of the service instance.                                                        |
+| service.name            | Logical name of the service.                                                                  |
+| service.namespace       | A namespace for `service.name`.                                                               |
+| service.version         | The version string of the service API or implementation.                                      |
+| telemetry.sdk.language  | The language of the telemetry SDK.                                                            |
+| telemetry.sdk.name      | The name of the telemetry SDK.                                                                |
+| telemetry.sdk.version   | The version string of the telemetry SDK.                                                      |
```

Comment:
I'd be inclined to drop the table for this package, an internal test utils package. But totally fine to have it in.

Reply:
Agree. I removed the table, but keep the message of semantic version, to keep consistent with other places.